submissions/david/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AI Izumo ChatBot</title>
+
+</head>
+<body>
+    <div class="navbar">NAV</div>
+
+    <div class="main-wrapper">
+        <div class="main" id="mainDiv"></div>
+        <div class="input" id="divInput"></div>
+    </div>
```

Comment:
Sorry, what I meant was to create the DOM in pure HTML as much as possible to:
- simplify the code
- improve performance

Something like it:
```suggestion
    <div class="navbar">NAV</div>

    <div class="main-wrapper">
        <div class="main" id="mainDiv"></div>
        <div class="input" id="divInput">
          <input type="text">
          <button class="btn-send">Send</button>
        </div>
    </div>
```

Then, you use the JS code where you absolutely need it:
- event handling
- list of elements generation
- ...